### PR TITLE
Ports waterworks from tg

### PR DESCRIPTION
**Pull request by BrumalLake**
## About The Pull Request
Ports [tgstation#52865](https://github.com/tgstation/tgstation/pull/52865)
Makes showers, sinks, and tanks plumbing enabled

- [x] Tanks
- [x] [tgstation#53579](https://github.com/tgstation/tgstation/pull/53579)
- [x] [tgstation#54236](https://github.com/tgstation/tgstation/pull/54236)
- [x] [tgstation#55895](https://github.com/tgstation/tgstation/pull/55895)
- [x] [tgstation#55894](https://github.com/tgstation/tgstation/pull/55894)
- [x] icons
- [ ] Figure out how to make plumbing ports face the right way
- [ ] Fix input pipes on sinks and showers not taking chems

Not doing hydrotrays, maybe if the rework ever gets ported

## Changelog
:cl:
add: Plumbing enabled sinks, showers, and tanks
/:cl:

### CODE REVIEW

**Comment by BrumalLake** [comment]
This bot only gets rid of 4 types of labels?

**Comment by Mat05usz** [comment]
The more the merrier 😳

File: code/game/objects/structures/shower.dm [hunk 1]
```diff
@@ -1,33 +1,49 @@
 #define SHOWER_FREEZING "freezing"
+#define SHOWER_FREEZING_TEMP 100
 #define SHOWER_NORMAL "normal"
+#define SHOWER_NORMAL_TEMP 300
 #define SHOWER_BOILING "boiling"
+#define SHOWER_BOILING_TEMP 400
+#define SHOWER_REACTION_MULTIPLIER 0.05
```
Comment (Archanial) [inline code/game/objects/structures/shower.dm]: remember to undef

Comment (BrumalLake) [reply code/game/objects/structures/shower.dm]: i did dw

**Comment by francinum** [comment]
I'm really not sure what to think of this featureset, primarily because it's honestly too little too late when it comes to justifying the existence and the current state of the plumbing network as a feature.

**Comment by BrumalLake** [comment]
> 
> 
> I'm really not sure what to think of this featureset, primarily because it's honestly too little too late when it comes to justifying the existence and the current state of the plumbing network as a feature.

I think some plumbing updates need to be ported first, this stuff for some reason won't connect or face the right direction and other things
Plus the current state of plumbing is pretty useless without multi layer devices

**Comment by BrumalLake** [comment]
This is really annoying to port, with how outdated hydroponics and plumbing are
I was mainly doing this for constructable showers so I'll just do that in a seperate PR

**Pull request closed without merging**